File: backend/src/main/java/com/livana/backend/pool/dto/PoolSummaryDto.java
package com.livana.backend.pool.dto;

import java.time.OffsetDateTime;

public record PoolSummaryDto(
    String onChainAddress,
    String title,
    String description,
    String region,
    String coverImageCid,
    long targetAmount,
    long totalDonated,
    long totalReleased,
    boolean isPaused,
    OffsetDateTime deployedAt
) {}

File: android/app/src/main/java/com/livana/app/core/network/dto/PoolSummaryDto.kt
package com.livana.app.core.network.dto

import kotlinx.serialization.Serializable

@Serializable
data class PoolSummaryDto(
    val onChainAddress: String,
    val title: String,
    val description: String,
    val region: String,
    val coverImageCid: String?,
    val targetAmount: Long,
    val totalDonated: Long,
    val totalReleased: Long,
    val isPaused: Boolean,
    val deployedAt: String,
)
